install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+# -*- sh-basic-offset: 4 -*-
```

Comment:
Is there any practical reason for setting *your* preferred Vim and Emacs settings in every bash script? Please don't tell me it's a "good practice".